edit/edit.go

```diff
@@ -90,6 +90,9 @@ func ShortenLabel(label string, pkg string) string {
 		}
 		return "@" + repo + "//" + labelPkg
 	}
+	if strings.HasPrefix(label, "@") && repo == rule {
```

Comment:
based on your test, this looks incomplete

Reply:
I've made some changes.

Reply:
it seems to me this simplifies as just
if  strings.HasPrefix(label, "@") && repo == rule && labelPkg == "" {
￼   return "@" + repo
￼ }

Reply:
This makes sense, thanks for feedback again.